js/__tests__/helper.test.js

```diff
@@ -22,6 +22,30 @@ describe('SentryCli helper', () => {
     expect(helper.getPath()).toMatch(pattern);
   });
 
+  describe('execute', () => {
+    test('execute with live=false returns stdout', async () => {
+      const output = await helper.execute(['--version'], false);
+      expect(output.trim()).toBe('sentry-cli DEV');
+    });
+
+    test('execute with live=true resolves without output', async () => {
+      // TODO (v3): This should resolve with a string, not undefined/void
```

Comment:
Please open an issue for this, if you have not already

Reply:
good point! done in https://github.com/getsentry/sentry-cli/issues/2606